pkg/observability/trace.go

```diff
@@ -28,7 +28,7 @@ func Start() error {
 		return nil
 	}
 
-	exporter, err := stdouttrace.New(stdouttrace.WithPrettyPrint())
+	exporter, err := otlptracehttp.New(context.TODO())
```

Comment:
I little nitpick-y but i think this should be `context.Background()` since I don't _think_ the intention is to eventually have `Start` function here accept a context

```suggestion
	exporter, err := otlptracehttp.New(context.Background())
```